pkgs/applications/version-management/sparkleshare/default.nix

```
@@ -55,20 +55,18 @@ stdenv.mkDerivation rec {
 
   postInstall = ''
     wrapProgram $out/bin/sparkleshare \
-        --set PATH ${symlinkJoin {
-          name = "mono-path";
-          paths = [
+        --prefix PATH ${lib.makeBinPath [
             bash
             coreutils
-            git
-            git-lfs
             glib
             mono
-            openssh
             openssl
-            xdg-utils
-          ];
-        }}/bin \
+          ]
+        } \
+        --suffix PATH ${lib.makeBinPath [
```

The build fails right now:

```suggestion
        --suffix PATH : ${lib.makeBinPath [
```